modules/swagger-codegen/src/main/java/io/swagger/codegen/InlineModelResolver.java

```diff
@@ -191,6 +191,16 @@ public void flatten(Swagger swagger) {
         }
     }
 
+    private void fixStringModel(ModelImpl m) {
+        if (m.getType() != null && m.getType().equals("string") && m.getExample() != null) {
+            String example = m.getExample().toString();
+            if (example.substring(0, 2).equals("\"") &&
+                    example.substring(example.length() - 2).equals("\"")) {
+                m.setExample(example.substring(2, example.length() - 2));
```

Comment:
@JFCote I added the example snippet to src/main/resources/petstore.yaml, and regenerated the code (from your branch). The [bug still occurs there](https://github.com/swagger-api/swagger-codegen/commit/947303bd7adab7d458919972357bd4accd082bd9#diff-7e2ada6ae234106c8b7784808b18326fR633). [Using 1 instead of 2 fixes it.](https://github.com/swagger-api/swagger-codegen/commit/c4be016d212dea0f63f380ccdd6f9d82cbec1e5a).

(Feel free to pick those commits [from my fork](https://github.com/StingrayDigital/swagger-codegen/compare/fix-5460...ePaul:5460-5778-test), or copy just the needed parts.)